### PR TITLE
Add missing messages

### LINKED ISSUE #2608: Missing messages
There are messages missing in messages.properties. For example /find and /send :( Please add all messages.
https://gyazo.com/1cdf6584b9c1db8e2abb6edf18746f1c

### PR DESCRIPTION
**Pull request by MrIvanPlays**
Closes #2608

### CODE REVIEW

File: module/cmd-alert/src/main/java/net/md_5/bungee/module/cmd/alert/CommandAlertRaw.java [hunk 1]
```diff
@@ -40,15 +40,15 @@ public void execute(CommandSender sender, String[] args)
                 }
                 if ( sender instanceof ProxiedPlayer )
                 {
-                    sender.sendMessage( new ComponentBuilder( "An error occurred while parsing your message. (Hover for details)" )
-                            .color( ChatColor.RED )
```
Comment (md-5) [inline module/cmd-alert/src/main/java/net/md_5/bungee/module/cmd/alert/CommandAlertRaw.java]: Why this change?

File: module/cmd-alert/src/main/java/net/md_5/bungee/module/cmd/alert/CommandAlertRaw.java [hunk 2]
```diff
@@ -23,7 +23,7 @@ public void execute(CommandSender sender, String[] args)
     {
         if ( args.length == 0 )
         {
-            sender.sendMessage( ChatColor.RED + "You must supply a message." );
+            sender.sendMessage( ProxyServer.getInstance().getTranslation( "message_needed" ) );
```
Comment (Mystiflow) [inline module/cmd-alert/src/main/java/net/md_5/bungee/module/cmd/alert/CommandAlertRaw.java]: I think for completeness it would be better if the default translation for this matches that of send_cmd_usage

File: module/cmd-find/src/main/java/net/md_5/bungee/module/cmd/find/CommandFind.java [hunk 3]
```diff
@@ -20,16 +18,16 @@ public void execute(CommandSender sender, String[] args)
     {
         if ( args.length != 1 )
         {
-            sender.sendMessage( new ComponentBuilder( "Please follow this command by a user name" ).color( ChatColor.RED ).create() );
+            sender.sendMessage( ProxyServer.getInstance().getTranslation( "username_needed" ) );
```
Comment (Mystiflow) [inline module/cmd-find/src/main/java/net/md_5/bungee/module/cmd/find/CommandFind.java]: Same as alertraw review

File: module/cmd-alert/src/main/java/net/md_5/bungee/module/cmd/alert/CommandAlert.java [hunk 4]
```diff
@@ -19,7 +19,7 @@ public void execute(CommandSender sender, String[] args)
     {
         if ( args.length == 0 )
         {
-            sender.sendMessage( ChatColor.RED + "You must supply a message." );
+            sender.sendMessage( ProxyServer.getInstance().getTranslation( "message_needed" ) );
```
Comment (Mystiflow) [inline module/cmd-alert/src/main/java/net/md_5/bungee/module/cmd/alert/CommandAlert.java]: Same as alertraw review

File: module/cmd-send/src/main/java/net/md_5/bungee/module/cmd/send/CommandSend.java [hunk 5]
```diff
@@ -69,21 +68,21 @@ public void execute(CommandSender sender, String[] args)
...
                 {
-                    sender.sendMessage( ChatColor.RED + "That player is not online" );
+                    sender.sendMessage( ProxyServer.getInstance().getTranslation( "user_not_online" ) );
                     return;
                 }
                 summon( player, target, sender );
             }
         }
-        sender.sendMessage( ChatColor.GREEN + "Successfully summoned player(s)" );
+        sender.sendMessage( ProxyServer.getInstance().getTranslation( "successfully_summoned" ) );
```
Comment (Mystiflow) [inline module/cmd-send/src/main/java/net/md_5/bungee/module/cmd/send/CommandSend.java]: Possible to add number as players as art in translation?

File: module/cmd-send/src/main/java/net/md_5/bungee/module/cmd/send/CommandSend.java [hunk 6]
```diff
@@ -69,21 +68,21 @@ public void execute(CommandSender sender, String[] args)
                 ProxiedPlayer player = ProxyServer.getInstance().getPlayer( args[0] );
                 if ( player == null )
                 {
-                    sender.sendMessage( ChatColor.RED + "That player is not online" );
+                    sender.sendMessage( ProxyServer.getInstance().getTranslation( "user_not_online" ) );
```
Comment (Mystiflow) [inline module/cmd-send/src/main/java/net/md_5/bungee/module/cmd/send/CommandSend.java]: User arg to add to translation msg?

File: proxy/src/main/java/net/md_5/bungee/command/CommandIP.java [hunk 7]
```diff
@@ -18,16 +17,16 @@ public void execute(CommandSender sender, String[] args)
...
         {
-            sender.sendMessage( ChatColor.RED + "Please follow this command by a user name" );
+            sender.sendMessage( ProxyServer.getInstance().getTranslation( "username_needed" ) );
             return;
         }
         ProxiedPlayer user = ProxyServer.getInstance().getPlayer( args[0] );
         if ( user == null )
         {
-            sender.sendMessage( ChatColor.RED + "That user is not online" );
+            sender.sendMessage( ProxyServer.getInstance().getTranslation( "user_not_online" ) );
```
Comment (Mystiflow) [inline proxy/src/main/java/net/md_5/bungee/command/CommandIP.java]: Attempted string as arg in translation?

File: proxy/src/main/java/net/md_5/bungee/command/CommandIP.java [hunk 8]
```diff
@@ -18,16 +17,16 @@ public void execute(CommandSender sender, String[] args)
     {
         if ( args.length < 1 )
         {
-            sender.sendMessage( ChatColor.RED + "Please follow this command by a user name" );
```
Comment (Mystiflow) [inline proxy/src/main/java/net/md_5/bungee/command/CommandIP.java]: See alertraw review

File: module/cmd-find/src/main/java/net/md_5/bungee/module/cmd/find/CommandFind.java [hunk 9]
```diff
@@ -20,16 +18,16 @@ public void execute(CommandSender sender, String[] args)
...
         } else
         {
             ProxiedPlayer player = ProxyServer.getInstance().getPlayer( args[0] );
             if ( player == null || player.getServer() == null )
             {
-                sender.sendMessage( new ComponentBuilder( "That user is not online" ).color( ChatColor.RED ).create() );
+                sender.sendMessage( ProxyServer.getInstance().getTranslation( "user_not_online" ) );
```
Comment (Mystiflow) [inline module/cmd-find/src/main/java/net/md_5/bungee/module/cmd/find/CommandFind.java]: Attempted string as arg?
